constant/skill/class.go

```diff
@@ -17,3 +17,277 @@ type gm struct {
 
 type superGm struct {
```

Comment:
Can delete this and the other un-exported classes

Reply:
Perfect, I was unsure if they were being use elsewhere already.